kolibri/plugins/learn/assets/src/vue/breadcrumbs/index.vue

```diff
@@ -1,38 +1,92 @@
 <template>
 
-  <nav class="nav" role="navigation" aria-label="You are here:">
-    <span class="parent">
-      <a v-link="rootLink">Explore</a> 
-    </span>
-    <span class="parent" v-for="crumb in crumbs">
-      <a v-link="crumbLink(crumb.id)">{{ crumb.title }}</a> 
-    </span>
-    <span class="current">
-      <span class="visuallyhidden">Current: </span>
-        <!-- TODO: Get current topic title -->
-    </span>
-  </nav>
+  <div>
+    <nav class="nav" role="navigation" aria-label="You are here:">
+      <span class="parent full-breadcrumbs" v-if="pageMode === $options.PageModes.EXPLORE && pageName !== $options.PageNames.EXPLORE_CONTENT && !isRoot">
+        <a v-link="exploreRoot">Explore</a>
+      </span>
+      <span v-if="pageName === $options.PageNames.LEARN_CONTENT">
+        <a v-link="learnRoot">
+          <span class='sep'>
+            <svg height="24" viewbox="0 0 24 24" width="24" xmlns="http://www.w3.org/2000/svg">
+              <path d="M0 0h24v24H0z" fill="none"></path>
+              <path d="M20 11H7.83l5.59-5.59L12 4l-8 8 8 8 1.41-1.41L7.83 13H20v-2z"></path>
+            </svg>
```

Comment:
See @radinamatic recent work here: https://github.com/learningequality/kolibri/pull/355/files

Those used text characters, so maybe it's all right now that we're using SVGs? If so I imagine it needs a ` role="presentation"` attribute.

Also, they can they be externalized using `<svg src="..."></svg>`?